sdk/python/feast/registry.py

```diff
@@ -399,6 +395,10 @@ def refresh(self):
         """Refreshes the state of the registry cache by fetching the registry state from the remote registry store."""
         self._get_registry_proto(allow_cache=False)
 
+    def teardown(self):
+        """Tears down all local and cloud resources."""
```

Comment:
Why does the registry tear down all local and cloud resources? Seems confusing. Wouldn't we only be removing all registry specific resources?

Reply:
Hm, `deployed` doesn't sound right to me - it sounds like there's a server involved. What about just `Tears down (removes) the registry.`?

Reply:
sgtm

Reply:
I just want to be clear that we aren't touching any real infrastructure. We are dealing with the registry here.